routes/routes.js

```diff
@@ -38,5 +38,31 @@ router.post('/user/answers/question/:questionNumber', (req, res) => {
     })
 })
 
+router.get('/user/answers/question/:questionNumber', (req, res) => {  
+    let pageSize = 3; // TODO: define it in more stadarized way 
```

Comment:
This could be a uppercase constant in the file in case any other routes need it.

Not sure the best name to use, maybe `ANSWER_HISTORY_LIMIT` ?